src/test/java/com/box/sdk/BoxTaskAssignmentTest.java

```diff
@@ -83,6 +84,36 @@ public void testGetTaskAssignmentInfoByIDSucceeds() throws IOException {
         Assert.assertEquals(fileID, assignmentInfo.getItem().getID());
         Assert.assertEquals(assignedToID, assignmentInfo.getAssignedTo().getID());
         Assert.assertEquals(assignedByID, assignmentInfo.getAssignedBy().getID());
+        Assert.assertEquals(status, assignmentInfo.getStatus());
+    }
+
+    @Test
+    @Category(UnitTest.class)
+    public void testSetTaskOnTaskAssignmentSucceeds() throws IOException {
+        String result = "";
+        final String assignmentID = "12345";
+        final String assignmentURL = "/task_assignments/" + assignmentID;
+        final String status = "Incomplete";
+        JsonObject taskObject = new JsonObject()
+                .add("status", status);
+
+        result = TestConfig.getFixture("BoxTask/CreateTaskAssignment201");
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.get(WireMock.urlPathEqualTo(assignmentURL))
+                .willReturn(WireMock.aResponse()
+                        .withHeader("Content-Type", "application/json")
+                        .withBody(result)));
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.put(WireMock.urlPathEqualTo(assignmentURL))
```

Comment:
It doesn't look like this is actually used in this test — can it be removed? Could we move it to a test that actually asserts about the PUT behavior?